index.html

```diff
@@ -0,0 +1,49 @@
gSS5xLG9KUq1Lc+j6yNdSFiFRzyLfSxEGkweU0lmaTl/yuJaHBI3GYXTA0sHIJkSsnhnonLEICXpdwc+VfENRyUPqjPTJFaETdtBa1GoaaC70qPtoEYJb/VyB2P6llIaeqR/tPFzAGxD4GCKpE7IWXzf+ipB00eJfl1JmmYoWQ6oetNHMdFnbPWhtwquvI1UIibPT5y66HD3VCAkqdZdG+NTbNY2yTxFeqxZVQusDtoX+JN1j8tD6hAe2CCt200e4cfR0y0f4UYT2NYHjBZ7EIuCQFSsoF4Q8EpFnJ3iHOe95r2MxEdiOLyR8hEN+sErgod7bQwgbXepC1OIjDJNnisaEBAgHjUk+NJBN1l0kLItwCH7X3B1bfjO8t1pMUtD5zJfF5GzvNE9DkIv9WaQSUh2/y+MyaRhF1X3yNI3ABcA18cFMzK9QC41dErYYQc1RiD5whm5NrDvp1g73aMKPMFrCL32kibam/Bxhu9GwLcBU0OT+Lqx1Hz/ojDA0hEv4yLZl+/34bcv10sO2pdYILq8NCVxrS+TnnKrGxhbOuWmfrbmg7gjnYzm9w9hmcgTCY9IQOlAowvWEmYjyODrUkyJo8QM8rO+z6K7Ftwg/iBrxDUV4/1W9VF9Qd4RfUITRR4rw4gM/7x+IqRBiHbLQ8hE28m5VTSTyvSGq+giz1m8HToi2mtiq6j9aEW4mRHkLIdpcMjXWt3reK7xgI8eKXvr2dxOudOm7UvXH5UpOzLe1tghmbOlYPite07QSNqoYjyE0o6cV1vrflSI815duE1JxJEtN/ODNTCNhhzEVGgjR4n2gs9MmJGTgUjRqIATE/eb/htAfEVMsT7W69CTU4rg6Ap6YmmZlv7eomtG8rzpwHUYcaCCEYCRa6q8RphLqinDa26ka5HOEEHNr8HwZi088P5YQhn36FlcJKSFYTsSIK6o+gRXSL733GUKUFotVMs0ycS8fS0jQ1Hza/vUqD/dKpKoowny2wUmGxU4+jqLUWEo7jKnQSGi1G6/VmV3lSPUFbEIe6WWJlefeZwiR9iDi1imh9pc3pfRogpRXVf35WF1chTDE22zP9Q6wzxFWzSZs+T77d62J0MVGbDvaeZjhLFktxINYUROLwzzxEWkmtApke8SBhxIS4pmrOtpq4nx8eTlhcQmfU4SR7ThWmzNjzNZLH5CHxHqDWVeEY4fn1acJ0ZMh+lsIbROEjVEjWgnLCqE1SxW1+3k/P6SUHojrOHbAec/zRNSInlQTpfo3ZDYRirD4oIBKgYR8Pq8QBuYMkjCv7AM75dn5IYSNUq0H8yeb7NKqCC8q7y0CGVPLEDZZ9gjCyM4J8Phm9zYFrwKYSBDHRtZnBl/2gKB5jsvz3KQE/Jsf1VKa6rPByagoJ4G9T+xaamJ3hM32ofDbkqIJ+xhvz/i8Pz+LKRx4VuvN6H1+jPB7JgmDfnI44+fRIZHzgG+f7AcI2zXsD3cCQp7N1kDlpwnDYrNYGeslyXp6GENdugG3kLmxshyNB1Dp+xn4i4iVO5GyH+NqXTrD1fbQXdtH2/GjnY8Xc0HdEe5OZc3mwkd4vupd2WoFF57i6/UsZV5WCFPrkoWP8Lp+sMPgEc9hmyL8gY9wY1yMSbVPk1vfJmj1EU61Iux1SPgVRZgYRbj2nG6uCF9N0qcU4e4JK/MintpUfa0IExIUyjmFQODExXXP2+z0Cb20yzzc6xEw8r9ACB/AAnmDr8166JrQtp8h1D0OPbLfnfFRXUUroVYTvd37aYBhX4Km4Lh47ZtomSCsIvB78xhC7X2J8xSrUIOCcFjfURGSdLfdbyJ+gQmfRXiDMBIIIe+lp5HPxdbHEoILzUYNebY3CVltNBE50RPhHj9HqN+fchVr+EhCiHrJhhP8xCoGrSCEWcondSWWmnjxeeJCPmA3CY3U5PqRCBr5SEIIUcKqv3MpVgpC5RR91q7OFiEV1eVGtQJfIATjz+HSImyoiP7E6oRiNn4mpx7bhAQtF7rqr2reFE2g+8mvtneL0IVbQvnn3RwZlf2hhDP+NCXbBkJqte9XKkbfaBWfIUQrPfblXoIdE0oxTBIKjVgGKJd6qYoiYe1oE1LWMEZoMt2NQE+9VqIaCNl9HPYLHvuyoDIPpXVBqCLFSkLKw8ur4B8VQtsqeZjCfdrvywy2/gyhfa+kF7S0Dghnc1lOVWuxByAFdbxNCH69uTXkv0lIpaMAEUN72LHoNA9fZb9NEhIP42es5lwJwrS+o9ZLoRNeaS8/Igwl4bUZvbQDwly9m1GEyF9Nh+rhFHppWilW3CxFeLqopFQIK584JD9DOFQqwweKcKqUInPBVh761WnP67I6tjD7EOFBq49lUgod6a4DwqXqRDQTApxQPCkP5eHIj5RG2gsaYmWw9UpMjjOhYvQ4hNpHyMyCEImd7KMttIrRAaFu08b4dDwe368c7qCZa9RwobA1K8JTaD9afYSbFeGyw/ZQRy6bgGswD8qhJnJPp6s+FCauEXORWAYF6S+XObyBC4ZKPu7rlOGMtwYpT7L2KZYQv4d1hoZ5XtsndQ1hu/PmdwnfbwiG35QTv7lDF6X08j2GjqyDPOyXfxVhcG9CD/fXt7d6oHVAOPy7PvHYAWGT2vuD9kv4jQM2ffnvB83/DxDeCgn05QMu5re3eqDF353i0mYO67X9VZUpaRqM/snxCC4mk9vb/cEZrP5Ew6v/enSsclHb6M/OX2aoQaO44wm+mm6cJe90ARn+yK3izy0tNhszyagYVmtKgopeUPl7+H7vK/AgGgCdtU0q+1Mjw8lG4U2KZc4HUMjt63saZvZD0vK++c+sh8uu8L5uZL+7+ziAoA3Gcl7eXkZYO4O9gV2YPQt7eRkYEx99FnYS9ioNdILjkasFypo1gkY73L7iLxNSFExHJTh+MDvUrCGGmxIAMhHss9XWjZYYU64YvBCzUtxfRp2M5P6m0eGv/dqv/dqvPdT+B/w8IbS9a88oAAAAAElFTkSuQmCC"/>
```

Comment:
This is a great image! Going forward if you are going to use an image like this you should include a copy of the image with the code.